front/src/Phaser/Game/GameScene.ts

```diff
@@ -1219,5 +1234,18 @@ export class GameScene extends ResizableScene implements CenterListener {
         mediaManager.removeTriggerCloseJitsiFrameButton('close-jisi');
     }
 
+    public startBBBMeeting(meetingId: string, clientURL: string): void {
+        this.connection.setSilent(true);
+        mediaManager.hideGameOverlay();
+        coWebsiteManager.loadCoWebsite(clientURL, iframe => {
+            iframe.allow = `microphone ${clientURL}; camera ${clientURL}`;
+        });
```

Comment:
Nice!

Reply:
Actually, I don’t think this is right and it’s probably why it’s not working on Firefox. It should be of the form `https://bbb.example.org/` and not `https://bbb.example.org/bigbluebutton/…` I believe `clientURL` needs to be stripped away of its path.